pkgs/development/python-modules/lox/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+
+  pathos,
+  pytestCheckHook,
+  pytest-benchmark,
+  pytest-mock,
+  pytest-runner,
+  setuptools,
+  sphinx-rtd-theme,
+  tqdm,
+}:
+
+buildPythonPackage rec {
+  pname = "lox";
+  version = "0.12.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "BrianPugh";
+    repo = "lox";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Iv3ZdfsvFLU6lhlH1n+eQ+KIrXESsnC1S2lVFnKFV08=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    pathos
+    sphinx-rtd-theme # upstream package lists this as a runtime dep
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-benchmark
+    pytest-mock
+    pytest-runner
```

Comment:
```suggestion
```
We use `pytestCheckHook` to run pytest, so `pytest-runner` is not needed.

Reply:
Without pytest-runner the build phase fails to create the wheel.

Reply:
I found it. Upstream is pinning "pytest-runner" into the setup_requires inside of setup.py. I've removed it